refinery/tool_manager/tests.py

```diff
@@ -869,6 +868,14 @@ def test_get_extra_directories_workflow_tool_def(self):
         with self.assertRaises(NotImplementedError):
             self.td.get_extra_directories()
 
+    def test_related_workflow_inactive_after_deletion(self):
+        self.create_workflow_tool_definition()
+        self.assertTrue(self.td.workflow.is_active)
+        self.td.delete()
+        self.assertFalse(
+            Workflow.objects.all()[0].is_active
+        )
```

Comment:
Not particularly interested in `self.td` other than what happens after it gets deleted.
Since `ToolDefinitions` now have a foreign key to `Workflows` I want to assert that Workflows are set to an inactive state after a ToolDefinition is removed, I just can't access the Workflow object through the definition post-deletion, hence the ORM stuff